.drone.yml

```diff
@@ -7,30 +7,29 @@ platform:
   arch: amd64
 
 steps:
-- name: validate-list
-  image: library/busybox:1.33.0
-  commands:
-  - grep -vE '^\s*(#|//)' images-list | sort -dc
-- name: mirror-images
-  image: rancher/dapper:v0.5.3
-  volumes:
-  - name: docker
-    path: /var/run/docker.sock
-  environment:
-    DOCKER_PASSWORD:
-      from_secret: docker_password
-    DOCKER_USERNAME:
-      from_secret: docker_username
-  commands:
-  - cat images-list | dapper
-  settings:
-    custom_dns: 1.1.1.1
-  when:
-    event:
-    - push
+  - name: validate-list
+    image: library/ubuntu:21.04
+    commands:
+      - grep -vE '^\s*(#|//)' images-list | sort -Vc
```

Comment:
The only significant changes are:
1. Added the flag `-V` to sort so that it does version sorting
2. Changed the image to ubuntu. The reason for this is that busybox and alpine have a different version of sort that doesn't sort the images in the same way and the validation fails. Ubuntu's sort seems to match my mac's sort and passes validation without problems, so it's a better choice considering most of us work in Macs or a debian based OS.

Reply:
What's with all the whitespace changes?

Reply:
The whitespace is just format/lint adjusting the indentation.